sandbox/OpenIddict.Sandbox.Avalonia.Client/OpenIddict.Sandbox.Avalonia.Client/OpenId/MauiHosting.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Microsoft.Maui.Hosting
+{
+    /// <summary>
+    /// Represents a service that is initialized during the application construction.
+    /// </summary>
+    /// <remarks>
+    /// This service is initialized during the MauiAppBuilder.Build() method. It is
+    /// executed once per application using the root service provider.
+    /// </remarks>
+    public interface IMauiInitializeService
```

Comment:
IMHO, it doesn't make a lot of sense to have MAUI interfaces in an Avalonia app 😕